### ISSUE TITLE
Extra `input` event in `onblur` event handler

### ISSUE THREAD

**Issue by machinek56** [dm4t2/vue-currency-input#42]
Hi!
I have a form with two inputs - min and max and an 'onInput' event handler, that calls an api to update data.

My issue is when i blurring  input, the plugin dispatches additional input event and my api is calling twice.
I have several checks if values from input not changed, but in some cases that don't work.

I'm using plugin as a component.

Here is my configuration:

```javascript
const pluginOptions = {
  globalOptions: {
    currency: null,
    distractionFree: false,
    decimalLength: 0
  }
}

Vue.use(VueCurrencyInput, pluginOptions)
```
Example of using component
```html
<currency-input type="tel" name="min"
       v-model="priceInput.min"
       :placeholder="allFacets.price[1].value | thousandSeparator"
       :data-min="allFacets.price[1].value"
       ref="minPrice"
       @input="handlePriceInput"
       autocomplete="off"
       class="digi-price-input"
     ></currency-input>
```

I notice in vue-dev-tools, that blurring input causes two 'input' events.
![image](https://user-images.githubusercontent.com/12989125/68321321-8f910700-00d2-11ea-9975-e09da78c92d6.png)

I think that in `distractionFree: false` mode is not necessarily dispatch input event in onBlur handler.
[line 55 in directive.js](https://github.com/dm4t2/vue-currency-input/blob/master/src/directive.js#L55)

Vue - 2.6.10
vue-currency-input - 1.11.3

PS: if you want, i can create a minimal reproduction of this.

**Comment by dm4t2** [comment]
Hi! You are right, the `input` event on blur is actually unnecessary. It was originally a workaround for the [Element Input](https://element.eleme.io/2.12/#/en-US/component/input), but with the current version it works anyway. 

I will remove it.

**Comment by dm4t2** [comment]
Fixed in release 1.11.4 🎉
